test_conformance/math_brute_force/unary_two_results.cpp

```diff
@@ -220,7 +220,17 @@ int TestFunc_Float2_Float(const Func *f, MTdata d, bool relaxedMode)
     else
         float_ulps = f->float_ulps;
 
-    if (relaxedMode) float_ulps = f->relaxed_error;
+    if (relaxedMode)
+    {
+        if (gIsEmbedded)
+        {
+            float_ulps = f->float_embedded_relaxed_ulps;
+        }
+        else
+        {
+            float_ulps = f->relaxed_error;
+        }
+    }
```

Comment:
The overall change looks good to me. It's a bit unfortunate that all the code formatting makes this harder to review than it should be.

I think we should introduce a utility function to get the allowed relative error for a given function. How about
```
float getAllowedUlpError(Func* f, bool relaxed);
```

Reply:
I agree, a utility function would be nice here. I'll put something together.

Reply:
I've added a function to do this but kept its use restricted to the necessary locations for this particular issue.

I will raise a new issue to track using it throughout the rest of the test, as well as supporting double ULPs.